mobilecoinofficial/desktop-wallet
# Test TransactionView

Soundtrack of this PR: [Alan Watts: Silence is key](https://www.youtube.com/watch?v=rJwVlmlyhqc)

### Motivation

Continued test coverage expansion for Desktop Wallet components, views, utils and hooks.

### In this PR

- Tests snapshot of component matches
- Verify `SendMobPanel` renders
- Verify `ReceiveMobPanel` renders
- Tabs update panel views on click 

### Testing
PASS  test/views/wallet/TransactionView/TransactionView.test.tsx (8.707 s)

File                  | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
----------------------|---------|----------|---------|---------|-------------------
  index.tsx                                    |     100 |      100 |     100 |     100 |                                                                               

Test Suites: 1 passed, 1 total
Tests:       3 passed, 3 total
Snapshots:   1 passed, 1 total
Time:        14.264 s
